script/entity_processor.lua

```diff
@@ -575,4 +574,38 @@ entity_proc.add_entities_to_chunks = function(build_type, entities, queue, event
     end
 end
 
+-------------------------------------------------------------------------------
+--  Utility functions
+-------------------------------------------------------------------------------
+
+-- function to recursively search for other biter nests around a nest
+---@param enemy LuaEntity
+---@param enemies_list table<uint32, LuaEntity>
+---@return table<uint32, LuaEntity>
+entity_proc.recursive_search = function(enemy, enemies_list)
```

Comment:
maybe rename this to reflect that this is just recursive searching for enemies?

Reply:
changed to recursive_enemy_search()